`orbit` v1.52.0 panics when auto-updates disabled

**fleetd version**: v1.52.0

<hr/>

### 💥  Actual behavior

`orbit` panics when auto-updates are disabled (fleetd package built with `--disable-updates` or environment/configuration set `ORBIT_DISABLE_UPDATES=true`).
Regression was introduced in: https://github.com/fleetdm/fleet/commit/8b77f0bd79a7e11653250b780cf3b419bb0a6a41

```sh
2026-02-20T09:25:49-08:00 INF running with auto updates disabled
2026-02-20T09:25:59-08:00 INF running with auto updates disabled
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x48 pc=0x101172ccc]

goroutine 1 [running]:
main.main.func2(0x14000187f40)
        github.com/fleetdm/fleet/v4/orbit/cmd/orbit/orbit.go:941 +0x33ac
github.com/urfave/cli/v2.(*Command).Run(0x14000396000, 0x14000187f40, {0x1400003e0a0, 0x1, 0x1})
        github.com/urfave/cli/v2@v2.27.7/command.go:276 +0x580
github.com/urfave/cli/v2.(*App).RunContext(0x14000535200, {0x101cf0cd0, 0x1028bfc60}, {0x1400003e0a0, 0x1, 0x1})
        github.com/urfave/cli/v2@v2.27.7/app.go:333 +0x4c8
github.com/urfave/cli/v2.(*App).Run(...)
        github.com/urfave/cli/v2@v2.27.7/app.go:307
main.main()
        github.com/fleetdm/fleet/v4/orbit/cmd/orbit/orbit.go:1556 +0x1014
2026-02-20T09:26:10-08:00 INF running with auto updates disabled
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x48 pc=0x10362accc]

goroutine 1 [running]:
```

--- a/orbit/changes/40200-auto-update-panic
+++ b/orbit/changes/40200-auto-update-panic
@@ -1,0 +1,1 @@
+* Fix orbit panic when auto-updates disabled

--- a/orbit/cmd/orbit/orbit.go
+++ b/orbit/cmd/orbit/orbit.go
@@ -604,6 +604,7 @@ func main() {
 		// it fetches osqueryd once as part of initialization.
 		var updater *update.Updater
 		var updateRunner *update.Runner
+		var osqueryVersion string
 		if !c.Bool("disable-updates") || c.Bool("dev-mode") {
 			updater, err := update.NewUpdater(opt)
 			if err != nil {
@@ -645,6 +646,7 @@ func main() {
 				if err == nil && version != "" {
 					log.Info().Msgf("Found osquery version: %s", version)
 					updateRunner.OsqueryVersion = version
+					osqueryVersion = version
 				}
 			}
 
@@ -693,6 +695,10 @@ func main() {
 			osquerydPath, err = updater.ExecutableLocalPath(constant.OsqueryTUFTargetName)
 			if err != nil {
 				log.Fatal().Err(err).Msgf("locate %s", constant.OsqueryTUFTargetName)
+			}
+			if v, err := update.GetVersion(osquerydPath); err == nil && v != "" {
+				log.Info().Msgf("Found osquery version: %s", v)
+				osqueryVersion = v
 			}
 			if c.Bool("fleet-desktop") {
 				if runtime.GOOS == "darwin" {
@@ -924,7 +930,7 @@ func main() {
 			}
 
 			options = append(options,
-				osquery.WithFlags(osquery.FleetFlags(updateRunner.OsqueryVersion, parsedURL)),
+				osquery.WithFlags(osquery.FleetFlags(osqueryVersion, parsedURL)),
 				osquery.WithFlags([]string{"--tls_server_certs", certPath}),
 			)
 		} else if fleetURL != "https://" {
@@ -938,7 +944,7 @@ func main() {
 			}
 
 			options = append(options,
-				osquery.WithFlags(osquery.FleetFlags(updateRunner.OsqueryVersion, parsedURL)),
+				osquery.WithFlags(osquery.FleetFlags(osqueryVersion, parsedURL)),
 			)
 
 			if certPath = c.String("fleet-certificate"); certPath != "" {
@@ -1090,7 +1096,7 @@ func main() {
 			hostIdentityCertificatePath = hostIdentityCredentials.CertificatePath
 
 			options = append(options,
-				osquery.WithFlags(osquery.FleetFlags(updateRunner.OsqueryVersion, proxy.ParsedURL)),
+				osquery.WithFlags(osquery.FleetFlags(osqueryVersion, proxy.ParsedURL)),
 
 				// This is overriding the previous set of --tls_server_certs in osquery.FleetFlags above.
 				osquery.WithFlags([]string{"--tls_server_certs", proxy.CertificatePath}),

